packages/astro/src/compiler/index.ts

```diff
@@ -108,7 +108,7 @@ export async function compileComponent(
 
   // return template
   let moduleJavaScript = `
-import fetch from 'node-fetch';
+import 'isomorphic-fetch';
```

Comment:
This is needed to actually polyfill `fetch`

Reply:
Hm. If I leave `node-fetch` this is what happens:

```
TypeError: Cannot read property 'bind' of undefined
```

It doesn’t work with `node-fetch`. Do you have any other suggestions for a fix?

Reply:
I *think* this is the same bug Matthew saw earlier this week trying to get the launch ready. I think it’s a snowpack/esinstall thing. Let me check out and take a look quickly

Reply:
I think you fixed it! LGTM